### PR TITLE
Introduce a 15s RPC deadline to all calls

### PR DESCRIPTION
**Pull request by erikjoh**
It's good practice to add deadlines to all RPCs and in some cases a server may even enforce that deadlines are present.
Similarly to the connection timeout of 5s already present in `grpcdebug`, this change adds a 15s deadline to all RPCs as well.
Making the deadline configurable probably isn't needed since 15s should be sufficient in most/all cases.

This change also adds the xDS resolver and upgrades some dependencies.

### CODE REVIEW

**Review by menghanl — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -154,6 +154,9 @@ import (
 	_ "github.com/envoyproxy/go-control-plane/envoy/type/tracing/v3"
 	_ "github.com/envoyproxy/go-control-plane/envoy/type/v3"
 	_ "github.com/envoyproxy/go-control-plane/envoy/watchdog/v3"
+
+	// Add the xDS resolver to allow resolving using a "xds:///" target.
+	_ "google.golang.org/grpc/xds"
```
Comment (lidizheng) [inline main.go:159]: Good catch!

**Review by lidizheng — APPROVED** [review]
Thanks, this is very helpful!

**Comment by erikjoh** [comment]
Thanks for having a look at this so quickly! :smile: 
Could we get a release with this change as well?

**Comment by lidizheng** [comment]
Done. The binaries will be uploaded shortly.

**Pull request merged**